DigitalLearningSolutions.Data/Models/CustomPrompts/CourseAdminFields.cs

```diff
@@ -1,18 +1,19 @@
-﻿namespace DigitalLearningSolutions.Data.Models.CustomPrompts
-{
-    using System.Collections.Generic;
-
-    public class CourseCustomPrompts
-    {
-        public CourseCustomPrompts(int customisationId, int centreId, List<CustomPrompt> courseAdminFields)
-        {
-            CustomisationId = customisationId;
-            CentreId = centreId;
-            CourseAdminFields = courseAdminFields;
-        }
-
-        public int CustomisationId { get; set; }
-        public int CentreId { get; set; }
-        public List<CustomPrompt> CourseAdminFields { get; set; }
-    }
-}
+﻿namespace DigitalLearningSolutions.Data.Models.CustomPrompts
+{
+    using System.Collections.Generic;
+
+    public class CourseAdminFields
+    {
+        public CourseAdminFields(int customisationId, int centreId, List<CustomPrompt> adminFields)
+        {
+            CustomisationId = customisationId;
+            CentreId = centreId;
+            AdminFields = adminFields;
+        }
+
+        public int CustomisationId { get; set; }
+        public int CentreId { get; set; }
+
+        public List<CustomPrompt> AdminFields { get; set; }
```

Comment:
Super minor, but is there a reason behind inconsistent spacing between these fields/properties?

Reply:
hmm no, I've evened it out